### DISCUSSION TITLE
installation

### DISCUSSION THREAD

**Discussion by nikitameena23** [iterate-ch/cyberduck#14481] [Q&A] [answered]
i installed the latest version of cyberduck software, after complete the installation process they show me the repair option next to the uninstall button i clicked on repair option after the repairing process my pc need to restart, then i again open the cyberduck they show me the same process again and again.
what will i do ??

![Uploading cyberduck.jpg…]()

**Comment by AliveDevil** [comment]
You didn't wait for the screenshot to upload completely, please edit your comment and reupload the screenshot.

**Comment by AliveDevil** [comment, marked as answer]
Additionally, please start Cyberduck from the Start Menu, instead of the application installer.
